cprover_bindings/src/goto_program/location.rs

```diff
@@ -81,6 +93,57 @@ impl Location {
         Location::Loc { file, function, line, col }
     }
 
+    /// Create a Property type Location
+    pub fn property_location<T, U>(
+        file: Option<U>,
+        function: Option<U>,
+        line: T,
+        col: Option<T>,
+        comment: U,
+        property_name: U,
+    ) -> Location
+    where
+        T: TryInto<u64>,
+        T::Error: Debug,
+        U: Into<InternedString>,
+    {
+        let file = file.unwrap().into();
+        let line = line.try_into().unwrap();
+        let col = col.map(|x| x.try_into().unwrap());
+        let function = function.intern();
+        let property_class = property_name.into();
+        let comment = comment.into();
+        Location::Property { file, function, line, col, comment, property_class }
+    }
+
+    /// Create a Property type Location from an already existing Location type
+    pub fn create_location<T: Into<InternedString>>(
```

Comment:
This function need a better name. `with_property`? Change the final `location` to an initial `&self`?

Reply:
Yup, will do